migration-guide.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Migrating to the Management API
+sidebar_label: Migration guide
+sidebar_position: 90
+description: Migration to the Management API from Partner API
+pagination_next: null
+pagination_prev: null
+---
+
+# Migration to the Management API from Partner API
+
+It is easy to migrate from the [Partner API](https://developer.vippsmobilepay.com/docs/APIs/partner-api/).
+Just change the following three endpoint paths.
+
+## Get merchant by organization number
+
+Replace
+[`GET:partner-api/v0/merchants/{orgno}`](https://developer.vippsmobilepay.com/api/partner#tag/Merchants/operation/getMerchant)
+with
+[`GET:management/v1/merchants/{orgno}/sales-units`](https://developer.vippsmobilepay.com/api/management/#tag/Merchants/operation/getMerchantSalesUnits).
```

Comment:
Should be [GET:/management/v1/merchants/{scheme}/{id}](https://developer.vippsmobilepay.com/api/management/#tag/Merchants/operation/getMerchant)